atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/SweepStrategyManagers.java

```diff
@@ -74,42 +73,34 @@ public void invalidateCaches(Set<TableReference> tableRefs) {
         };
     }
 
-    public static SweepStrategyManager createFromSchema(Schema schema) {
-        return tableRef -> {
-            TableMetadata tableMeta = Preconditions.checkNotNull(
-                    schema.getAllTablesAndIndexMetadata().get(tableRef),
-                    "unknown table",
-                    SafeArg.of("tableRef", tableRef));
-            return SweepStrategy.from(tableMeta.getSweepStrategy());
-        };
-    }
-
     public static SweepStrategyManager fromMap(final Map<TableReference, SweepStrategy> map) {
         return tableRef ->
                 Preconditions.checkNotNull(map.get(tableRef), "unknown table", SafeArg.of("tableRef", tableRef));
     }
 
-    public static SweepStrategyManager completelyConservative() {
```

Comment:
I assumed this was also OK to change, since gotham-core is the only place using it and should be ok to pass a KVS here in their context?